product_docs/docs/biganimal/release/administering_cluster/notifications.mdx

```diff
@@ -54,18 +54,18 @@ Users in the following roles can view the notifications:
 - Project owners/editors can view the project notifications.
 - User can view their own account notifications. 
 
-Each notification indicates the level and project it belongs to for the user having multiple roles in BigAnimal.
+For the user having multiple roles in BigAnimal, each notification indicates the level and project it belongs to.
 
 Select the bell at the top of your BigAnimal portal to view the in-app notifications. By selecting the bell, you can read the notification, mark it as unread, and archive it.
 
 To view the email notifications, check the inbox of your configured email addresses.
 
-## Manage notifications
+## Managing notifications
```

Comment:
To be consistent with the other heads. (I updated the links to this section as well.)